github/actions_hosted_runners.go

```diff
@@ -0,0 +1,333 @@
+	u := fmt.Sprintf("orgs/%v/actions/hosted-runners", org)
+	req, err := s.client.NewRequest("POST", u, request)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	hostedRunner := new(HostedRunner)
+	resp, err := s.client.Do(ctx, req, hostedRunner)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return hostedRunner, resp, nil
+}
+
+// HostedRunnerImageSpecs represents the details of a github-hosted runner image.
+type HostedRunnerImageSpecs struct {
+	ID          string `json:"id"`
+	Platform    string `json:"platform"`
+	SizeGB      int    `json:"size_gb"`
+	DisplayName string `json:"display_name"`
+	Source      string `json:"source"`
+}
+
+// HostedRunnerImages represents the response containing the total count and details of runner images.
+type HostedRunnerImages struct {
+	TotalCount int                       `json:"total_count"`
+	Images     []*HostedRunnerImageSpecs `json:"images"`
+}
+
+// GetHostedRunnerGithubOwnedImages gets the list of GitHub-owned images available for github-hosted runners for an organization.
+//
+// GitHub API docs: https://docs.github.com/rest/actions/hosted-runners#get-github-owned-images-for-github-hosted-runners-in-an-organization
+//
+//meta:operation GET /orgs/{org}/actions/hosted-runners/images/github-owned
+func (s *ActionsService) GetHostedRunnerGithubOwnedImages(ctx context.Context, org string) (*HostedRunnerImages, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/actions/hosted-runners/images/github-owned", org)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	hostedRunnerImages := new(HostedRunnerImages)
+	resp, err := s.client.Do(ctx, req, hostedRunnerImages)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return hostedRunnerImages, resp, nil
+}
+
+// GetHostedRunnerPartnerImages gets the list of partner images available for github-hosted runners for an organization.
+//
+// GitHub API docs: https://docs.github.com/rest/actions/hosted-runners#get-partner-images-for-github-hosted-runners-in-an-organization
+//
+//meta:operation GET /orgs/{org}/actions/hosted-runners/images/partner
+func (s *ActionsService) GetHostedRunnerPartnerImages(ctx context.Context, org string) (*HostedRunnerImages, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/actions/hosted-runners/images/partner", org)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	hostedRunnerImages := new(HostedRunnerImages)
+	resp, err := s.client.Do(ctx, req, hostedRunnerImages)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return hostedRunnerImages, resp, nil
+}
+
+// HostedRunnerPublicIPLimits represents the static public IP limits for github-hosted runners.
+type HostedRunnerPublicIPLimits struct {
+	PublicIPs *PublicIPUsage `json:"public_ips"`
+}
+
+// PublicIPUsage provides details of static public IP limits for github-hosted runners.
+type PublicIPUsage struct {
+	Maximum      int `json:"maximum"`
+	CurrentUsage int `json:"current_usage"`
```

Comment:
Should these be `int64` ?
Could you please add a comment or update the comment on line 182 describing what the units are for these fields?